cellpack/autopack/Compartment.py

```diff
@@ -2400,7 +2399,6 @@ def getSurfaceInnerPoints_sdf(
         Only compute the inner point. No grid.
         This is independant from the packing. Help build ingredient sphere tree and representation
         """
-        print("beforea import")
         from autopack.Environment import Grid
```

Comment:
We could do the import at the top of the file for consistency. Or maybe local import is fine if it's only getting used in this one file.